.github/workflows/upgrade-develop-testing.yml

```diff
@@ -83,6 +84,36 @@ jobs:
       new-version: develop
       multisite: ${{ matrix.multisite }}
 
+  # Run a limited set of upgrade tests for the current branch on forks.
+  upgrade-tests-develop-forks:
+    name: Upgrade from ${{ matrix.wp }}
+    uses: ./.github/workflows/reusable-upgrade-testing.yml
+    if: ${{ github.repository != 'WordPress/wordpress-develop' }}
+    needs: [ build ]
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ 'ubuntu-24.04' ]
+        php: [ '7.2', '8.4' ]
```

Comment:
Noting that this should be 7.4instead of 7.2 if https://github.com/WordPress/wordpress-develop/pull/9181 lands first